www-client/qutebrowser/qutebrowser-9999.ebuild

```diff
@@ -30,10 +31,14 @@ RDEPEND="${COMMON_DEPEND}
 	>=dev-python/jinja-2.8[${PYTHON_USEDEP}]
 	>=dev-python/pygments-2.1.3[${PYTHON_USEDEP}]
 	>=dev-python/pypeg2-2.15.2[${PYTHON_USEDEP}]
-	dev-python/PyQt5[${PYTHON_USEDEP},declarative,gui,network,printsupport,webkit,widgets]
+	dev-python/PyQt5[${PYTHON_USEDEP},declarative,gui,network,printsupport,webengine?,webkit?,widgets]
 	dev-python/sip[${PYTHON_USEDEP}]
 	>=dev-python/pyyaml-3.12[${PYTHON_USEDEP}]
 	gstreamer? ( dev-qt/qtwebkit:5[gstreamer] )
+	webengine? (
```

Comment:
Whoops. It's a new requirement as of February 17, sorry, I was wrong.
https://github.com/qutebrowser/qutebrowser/commit/a86170f45dbc56b62f33709f33a258a077afe6f4

Also, technically PyQt 5.7 also works (though we don't have it in the tree)
https://github.com/qutebrowser/qutebrowser/commit/71f7b01ea7200a9a5786c76677ecd2ada1ff5d18

Carry on. :P